assets/sass/base/_defaults.scss

```diff
@@ -37,6 +37,8 @@
 	}
 
 	svg:not(:root) {
+		/* stylelint-disable-next-line sh-waqar/declaration-use-variable */
+		color: inherit;
```

Comment:
Pretty sure this caused a bunch of VRT failures:

<img width="1133" alt="Screenshot 2021-03-18 at 19 17 08" src="https://user-images.githubusercontent.com/90871/111684365-8b284280-881e-11eb-808a-37c2a2272425.png">